pkgs/development/libraries/embree/default.nix

```diff
@@ -24,8 +24,14 @@ stdenv.mkDerivation rec {
     "-DEMBREE_RAY_MASK=ON"
   ];
 
+  patchPhase = ''
```

Comment:
Please put this in the `postPatch` hook.  At the moment, it overrides `postPatch` and this breaks the blender build.